### ISSUE TITLE
the variant can not see

### ISSUE THREAD

**Issue by sofu456** [Unity-Technologies/vscode-unity-debug#159]
the variant in StartCoroutine cannot see the value.and i cannot find the ms-vscode.csharp plugin,because the csharp i install is ms-dotnettools.csharp, is the unity plugin need update?the depends is out of date?

**Comment by miniwolf** [comment]
Your last issue has been fixed and has been pushed live: #152

**Comment by sofu456** [comment]
thank you for your mention,but another question the variant in startcoroutine can not see it in watch variant window,or mouse hover.mybe a bug

**Comment by miniwolf** [comment]
Yes I know that to be the case that it is a bug. Thank you for reporting this.
